requirements/base.in

```diff
@@ -10,4 +10,5 @@ jsonfield2
 django-fernet-fields
 django-config-models
 
+edx-tincan-py35
```

Comment:
Since this will be running in python 3.8 and later mostly, we can drop `edx-tincan-py35` support and simply use the upstream tincan library.

Reply:
done